tests/fixtures/optimizer/optimizer.sql

```diff
@@ -276,3 +276,54 @@ SELECT /*+ COALESCE(3),
 FROM `x` AS `x`
 JOIN `y` AS `y`
   ON `x`.`b` = `y`.`b`;
+
```

Comment:
Do we need these tests in both fixture files?

Redundant tests can be a pain to fix.